eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -410,10 +410,16 @@ function VerifyPackages($pkgRepository, $artifactLocation, $workingDirectory, $a
         exit(1)
       }
 
+      $tag = if ($parsedPackage.packageId) {
+        "$($parsedPackage.packageId)_$($parsedPackage.PackageVersion)"
+      } else {
+        $parsedPackage.PackageVersion
```

Comment:
Does this mean that c will not have a packageid? I get we plan to ship the entire thing as one but I do still think it is worth having a packageid for it so that later if we decide to ship different subsets of it we can be clear what parts we are shipping

Reply:
At this point we are automating the release processes that are in place but we should review this before GA of C. I've got it tracked here: https://github.com/Azure/azure-sdk-tools/issues/642